api/controllers/game/untargeted-one-off.js

```diff
@@ -28,7 +28,7 @@ module.exports = function (req, res) {
                     if (game.scrap.length < 1)
                       {
                         return Promise.reject({
-                          message: 'You can only play a 3 as a one-off, if there are cards in the scrap pile',
+                          message: 'game.snackbar.oneOffs.three.scrapIsEmpty',
```

Comment:
untargeted oneoff was not using i18n keys for its errors and had duplicate, hard-coded english errors for things that the /seven/untargetedOneOff endpoint already had in i18n. I consolidated them into the i18n version